c/str/py_str.cc

```diff
@@ -29,6 +29,8 @@ static PKArgs args_split_into_nhot(
 
 static oobj split_into_nhot(const PKArgs& args) {
   DataTable* dt = args[0].to_datatable();
+  if (dt == nullptr) return py::None();
```

Comment:
what about the case when the function is called without any arguments? we should probably have such test too

Reply:
Sure, fixed this one too